src/test/java/org/opensearch/flowframework/transport/WorkflowRequestResponseTests.java

```diff
@@ -153,69 +153,10 @@ public void testWorkflowRequestWithParams() throws IOException {
         assertEquals("bar", streamInputRequest.getParams().get("foo"));
     }
 
-    public void testWorkflowRequestWithUseCase() throws IOException {
```

Comment:
Should these tests be fully removed, or just remove the parts related to the use case and params?  

Since that particular method of provisioning produces a different code path, it's useful to make sure the existing params match the expected